Sources/SwiftInspectorVisitors/FileVisitor.swift

```diff
@@ -75,6 +75,7 @@ public final class FileVisitor: SyntaxVisitor {
       fileInfo.appendEnums(extensionVisitor.innerEnums)
       fileInfo.appendClasses(extensionVisitor.innerClasses)
       fileInfo.appendStructs(extensionVisitor.innerStructs)
+      fileInfo.appendTypealiases(extensionVisitor.innerTypealiases)
```

Comment:
We weren't propagating typealiases found within extensions to the file visitor. There are tests for this now.